src/schema/v2/me/conversation/conversationEventConnection/conversationMessageEvents.ts

```diff
@@ -0,0 +1,67 @@
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { get } from "lodash"
+
+const lastMessageId = (conversation) => {
+  return get(conversation, "_embedded.last_message.id")
+}
+
+// Prepare the incoming args by adding a `before: ${lastMessageId}` if it is missing.
+export const prepareConversationArgs = (conversation, args) => {
+  const conversationArgs = { ...args }
```

Comment:
No need to create intermediate object. Can just do `conversationArgs = args`, or even better just use `args` vs creating intermediate variables